shared/src/commonMain/kotlin/org/jetbrains/kotlinconf/View.kt

```diff
@@ -93,7 +93,8 @@ fun Session.asSessionCard(
         tags = tags ?: emptyList(),
         startsInMinutes = (startsAt - now).inWholeMinutes.toInt().let { diff ->
             if (diff in 1..30) diff else null
-        }
+        },
+        videoUrl = "https://www.youtube.com/watch?v=Ar73Axsz2YA",
```

Comment:
could you tell me if the link will be static?

Reply:
Oops! Thanks for catching this, it was my temporary change for testing. It should be mapped from the session coming from the backend (I know we don't send this field yet, this is preparation for later).

Fixed the commit now.